test/resources/layout-shared.css

```diff
@@ -25,7 +25,8 @@ body {
 }
 
 .input-playground {
-  width: 250px;
+  width: 100%;
```

Comment:
This is the default input style, which I think makes sense to have here too; without it, playground inputs with "limited" width extend way beyond their containers.